### PR TITLE
Install virtual directory under applications

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Carbon.IIS/Functions/Install-CIisAppPool.ps1 [hunk 1]
```diff
@@ -4,48 +4,58 @@ function Install-CIisAppPool
...
 
-    An app pool can run as several built-in service accounts, by passing one of them as the value of the `ServiceAccount` parameter: `NetworkService`, `LocalService`, or `LocalSystem`  The default is `ApplicationPoolIdentity`, which causes IIS to create and use a custom local account with the name of the app pool.  See [Application Pool Identities](http://learn.iis.net/page.aspx/624/application-pool-identities/) for more information.
-
-    To run the app pool as a specific user, pass the credentials with the `Credential` parameter. (In some versions of Carbon, there is no `Credential` parameter, so use the `UserName` and `Password` parameters instead.) The user will be granted the `SeBatchLogonRight` privilege.
-
-    If an existing app pool exists with name `Name`, it's settings are modified.  The app pool isn't deleted.  (You can't delete an app pool if there are any websites using it, that's why.)
+    An app pool can run as several built-in service accounts, by passing one of them as the value of the
+    `ServiceAccount` parameter: `NetworkService`, `LocalService`, or `LocalSystem`  The default is
```
Comment (decoyjoe) [inline Carbon.IIS/Functions/Install-CIisAppPool.ps1]: Missing a period between `LocalSystem` and "The"

File: Carbon.IIS/Functions/Install-CIisAppPool.ps1 [hunk 2]
```diff
@@ -54,25 +64,25 @@ function Install-CIisAppPool
...
+
         # Use the classic pipeline mode, i.e. don't use an integrated pipeline.
         [switch] $ClassicPipelineMode,
-        
+
         # Enable 32-bit applications.
         [switch] $Enable32BitApps,
-        
+
         # Run the app pool under the given local service account.  Valid values are `NetworkService`, `LocalService`, and `LocalSystem`.  The default is `ApplicationPoolIdentity`, which causes IIS to create a custom local user account for the app pool's identity.  The default is `ApplicationPoolIdentity`.
```
Comment (decoyjoe) [inline Carbon.IIS/Functions/Install-CIisAppPool.ps1]: `ApplicationPoolIdentity` is not one of the valid values for `$ServiceAccount`.

Also, `The default` is repeated twice in two separate sentences.

File: Carbon.IIS/Functions/Install-CIisAppPool.ps1 [hunk 3]
```diff
@@ -4,48 +4,58 @@ function Install-CIisAppPool
...
-
-    To run the app pool as a specific user, pass the credentials with the `Credential` parameter. (In some versions of Carbon, there is no `Credential` parameter, so use the `UserName` and `Password` parameters instead.) The user will be granted the `SeBatchLogonRight` privilege.
-
-    If an existing app pool exists with name `Name`, it's settings are modified.  The app pool isn't deleted.  (You can't delete an app pool if there are any websites using it, that's why.)
+    An app pool can run as several built-in service accounts, by passing one of them as the value of the
+    `ServiceAccount` parameter: `NetworkService`, `LocalService`, or `LocalSystem`  The default is
+    `ApplicationPoolIdentity`, which causes IIS to create and use a custom local account with the name of the app pool.
```
Comment (decoyjoe) [inline Carbon.IIS/Functions/Install-CIisAppPool.ps1]: `ApplicationPoolIdentity` is not one of the valid values for `$ServiceAccount`.

**Review by decoyjoe — APPROVED** [review]

File: Carbon.IIS/Functions/Install-CIisAppPool.ps1 (hunk 2 above)
Comment (splatteredbits) [reply Carbon.IIS/Functions/Install-CIisAppPool.ps1]: It's the default that IIS uses when creating an app pool, i.e. if they don't provide an identity, it gets set to ApplicationPoolIdentity. I rephrased this to make it a little clearer:

```markdown
If no service account or identity is
    provided, `Install-CIisAppPool` sets the app pool to run as `ApplicationPoolIdentity`, a principal that IIS creates
    and manages. See [Application Pool Identities](http://learn.iis.net/page.aspx/624/application-pool-identities/) for
    more information on these types of accounts.
```

**Pull request merged**